xbmc/cores/VideoPlayer/VideoRenderers/windows/RendererDXVA.cpp

```diff
@@ -150,35 +132,54 @@ bool CRendererDXVA::Configure(const VideoPicture& picture, float fps, unsigned o
       if (settings && settings->GetBool(CSettings::SETTING_VIDEOPLAYER_USESUPERRESOLUTION) &&
           CProcessorHD::IsSuperResolutionSuitable(picture))
       {
-        tryVSR = true;
+        m_tryVSR = true;
       }
     }
 
-    // create processor
-    m_processor = std::make_unique<DXVA::CProcessorHD>();
-    if (m_processor->PreInit() && m_processor->Open(picture) &&
-        m_processor->IsFormatSupportedInput(dxgi_format))
+    m_enumerator = std::make_shared<DXVA::CEnumeratorHD>();
+    if (m_enumerator->Open(picture.iWidth, picture.iHeight, dxgi_format))
     {
-      m_intermediateTargetFormat = CalcIntermediateTargetFormat(picture, tryVSR);
-      if (m_processor->SetOutputFormat(m_intermediateTargetFormat))
+      const auto settings = CServiceBroker::GetSettingsComponent()->GetSettings();
+      const auto setting = DX::Windowing()->SETTING_WINSYSTEM_IS_HDR_DISPLAY;
+      const bool systemUsesHDR =
+          (settings ? settings->GetBool(setting) && DX::Windowing()->IsHDRDisplay() : false) ||
+          DX::Windowing()->IsHDROutput();
+
+      if (CServiceBroker::GetLogging().IsLogLevelLogged(LOGDEBUG) &&
+          CServiceBroker::GetLogging().CanLogComponent(LOGVIDEO))
+      {
+        m_enumerator->LogSupportedConversions(
+            dxgi_format, CProcessorHD::AvToDxgiColorSpace(DXVA::DXGIColorSpaceArgs(picture)),
+            m_intermediateTargetFormat);
```

Comment:
When code reach this point  `m_intermediateTargetFormat` is not yet initialized, then in `LogSupportedConversions` the list of conversions not indicates which is picked

Reply:
Ah yes, I took a note to remove that parameter, which doesn't make sense anymore, and forgot to apply it. It may come back in another refactor.

Reply:
done, removed.